src/index.css

```diff
@@ -23,7 +23,7 @@
         color: white;
     }
 
-    .table-cell {
-        @apply w-80
+    .todo-text {
+        @apply w-40 text-ellipsis overflow-hidden whitespace-nowrap
```

Comment:
there is a nice class in tailwind `truncate` which implements these 3 (except the width)

Reply:
Worked great thank you!